alembic/operations/base.py

```diff
@@ -127,7 +127,6 @@ def %(name)s%(args)s:
                     "args": args,
                     "apply_kw": apply_kw,
                     "doc": fn.__doc__,
-                    "meth": fn.__name__,
```

Comment:
**mike bayer** (zzzeek) wrote:

this is fine though it's preferable to keep a commit to just one thing

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/1970